src/Shared/Microsoft.DotNet.Kusto/KustoQueryOption.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+namespace Microsoft.DotNet.Kusto
+{
+    public class KustoQueryOption
```

Comment:
Why not make this a record?

Reply:
What do you mean by "record"?

Reply:
`public record KustoQueryOption(string Name, object Value)`, that can be the entire type

Reply:
https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/records

Reply:
They're like classes, but allow for value-based equality, which is kind of fancy.